docs/consensus.qbk

```diff
@@ -0,0 +1,651 @@
+
+Lastly, if our node were not in the proposing mode, it would not include its own
+vote and just take the majority (>50%) position of its peers. In this example,
+our node would maintain its position of transactions 1, 2 and 3.
+
+['Checking Consensus]
+
+After updating its position, the node checks for supermajority agreement with
+its peers on its current position.  This agreement is of the exact transaction
+set, not just the support of individual transactions. That is, if our position
+is a subset of a peer's position, that counts as a disagreement. Also recall
+that effective close time agreement allows a supermajority of participants
+agreeing to disagree.
+
+Consensus is declared when the following 3 clauses are true:
+
+* `LEDGER_MIN_CONSENSUS` time has elapsed in the establish phase
+* At least 75% of the prior round proposers have proposed OR this establish
+  phase is `LEDGER_MIN_CONSENSUS` longer than the last round's establish phase
+* =minimumConsensusPercentage= of ourself and our peers share the same position
+
+The middle condition ensures slower peers have a chance to share positions, but
+prevents waiting too long on peers that have disconnected. Additionally, a node
+can declare that consensus has moved on if =minimumConsensusPercentage= peers
+have sent validations and moved on to the next ledger. This outcome indicates
+the node has fallen behind its peers and needs to catch up.
+
+If a node is not proposing, it does not include its own position when
+calculating the percent of agreeing participants but otherwise follows the above
+logic.
+
+['Accepting Consensus]
+
+Once consensus is reached (or moved on), the node switches to the =Accept= phase
+and signals to the implementing code that the round is complete. That code is
+responsible for using the consensus transaction set to generate the next ledger
+and calling =startRound= to begin the next round.  The implementation has total
+freedom on ordering transactions, deciding what to do if consensus moved on,
+determining whether to retry or abandon local transactions that did not make the
+consensus set and updating any internal state based on the consensus progress.
+
+[endsect] [/Consensus Overview]
+
+[section Consensus Type Requirements]
+
+[*Should I delete this section?]
+
+The consensus type requirements are given below as minimal implementation stubs.
+Actual implementations would augment these stubs with members appropriate for
+managing the details of transactions and ledgers within the larger application
+framework.
+
+[heading Transaction]
+The transaction type =Tx= encapsulates a single transaction under consideration
+by consensus.
+
+```
+struct Tx
+{
+   using ID = ...;
+   ID const & id() const;
+
+   //... implementation specific
+};
+```
+
+[heading Transaction Set]
+The transaction set type =TxSet= represents a set of =Tx=s that are collectively
+under consideration by consensus. A =TxSet= can be compared against other =TxSet=s
```

Comment:
As above, the `=Tx=s` and `=TxSet=s` don't get interpreted correctly here. I don't have a suggestion on how to rephrase.